docs/content/en/templates/sitemap-template.md

```diff
@@ -48,7 +51,7 @@ This template respects the version 0.9 of the [Sitemap Protocol](http://www.site
 ```xml
 <urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9"
   xmlns:xhtml="http://www.w3.org/1999/xhtml">
-  {{ range .Data.Pages }}
+  {{ range where .Pages ".Sitemap.Exclude" false }}
```

Comment:
Went with `.Pages` instead of `.Data.Pages` because Hugo Docs says they are the same but  the former is the preferred convention.